app/.idea/gradle.xml

```diff
@@ -11,6 +11,7 @@
           <set>
             <option value="$PROJECT_DIR$" />
             <option value="$PROJECT_DIR$/app" />
+            <option value="$PROJECT_DIR$/joysticklibrary" />
```

Comment:
Why is the joystick library stored in the master directory and not in the project libs directory?

Reply:
Also, what library is it? We should include a reference to it in our readme file.

Reply:
It can be deleted. I planned to use this library but later didn't use at all. Also in manifest.xml and settings.gradle file, joysticklibrary is useless.

Reply:
Alright, if you delete it in the next push, this conversation can be resolved :)